public/js/data.js

```diff
@@ -1,73 +1,37 @@
 if (sessionStorage.getItem('loggedIn') != "true") {location.href = "/admin-login";}
 
 $(document).ready( function () {
-
-    /* Levels chart */
-    var levZero = 0;
-    var levOne = 0;
-    var levTwo = 0;
-    var levThree = 0;
-
-    /* Store Type chart */
-    let small = 0;
-    let medium = 0;
-    let large = 0;
-    let convenience = 0;
-
+    google.charts.load('current', {'packages':['corechart']});
+    
     levelArray = null;
     typeArray = null;
-
-
-    if (!firebase.apps.length) {
-        firebase.initializeApp(config);
-    }
-
-    google.charts.load('current', {'packages':['corechart']});
     
-    // Creates connection to database.
-    var db = firebase.database();
-
-    // Links to head of database.
-    var ref = db.ref("live_weller");
-
-    // Links to markets list.
-    var marketsRef = ref.child("markets");
-
-
-    // Call async function
-    callFirebase(loadCharts);
-
-
-
-    // Start of callback
-    function loadCharts() {
+    $.post('/data', {type: "general"}, function(data) {
         levelArray = [
```

Comment:
with the change I proposed below, this would become
```javascript
levelArray = [ [ "Market Level", "Number of Markets" ] ];
for (let i = 0; i < data.levels.length; i++) {
    levelArray.push( [ "Level " + i, data.levels[i] ] );
}
```